apps/backend/src/iot-devices/application/use-cases/provision-device/provision-device.ts

```
@@ -0,0 +1,103 @@
+    const thingResult = await this.awsIot.createThing(
+      thingName,
+      deviceClass,
+      serialNumber,
+      thingTypeName,
+    );
+    if (thingResult.isFailure()) return thingResult;
+
+    const groupResult = await this.awsIot.addThingToThingGroup(thingName, thingGroupName);
+    if (groupResult.isFailure()) {
+      const r = await this.awsIot.deleteThing(thingName);
+      if (r.isFailure()) this.logger.warn({ msg: "Cleanup failed: deleteThing", thingName, error: r.error });
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)